format/src/_matbin.rs

```diff
@@ -1,12 +1,14 @@
 use std::io::{self, SeekFrom};
 
 use byteorder::{ReadBytesExt, LE};
+use thiserror::Error;
 
 use crate::io_ext::ReadFormatsExt;
 
-#[derive(Debug)]
+#[derive(Debug, Error)]
 pub enum MatbinError {
-    IO(io::Error),
+    #[error("Could not copy bytes {0}")]
+    IO(#[from] io::Error),
```

Comment:
`Io`

Reply:
Renamed in 5a8254ca826fda4d5c084f43e91449126f675289